### ISSUE TITLE
Requests show for all users, except admin

### ISSUE THREAD

**Issue by my95z34** [tidusjar/PlexRequests.Net#177]
If this is a bug report please make sure you have filled the following in:
(If it's not a bug and a feature request then just remove the below)

#### Applicable Logs (from `/logs/` directory or the Admin page): 

Nothing shows up in the logs regarding this issue.

#### Reproduction Steps:

Whenever I am logged in to my admin account, nothing in the Movie request page shows up.  If I log out of the admin account and am just logged in as a normal user, everything shows up just fine.  This is on multiple devices.  Seen here: http://imgur.com/a/gXLQz

I do have it set to auto approve movies.  Not sure if that makes any difference.  This never used to happen, though has been happening for the past couple PR versions.

**Comment by tidusjar** [comment]
Have you tried clearing the browser cache? Also have you tried changing the filters when admin to see if anything happens?

**Comment by my95z34** [comment]
I've done the shift + refresh. Not entirely clearing cache. Just found it
interesting that it happens on more than just one device. And when one
works, they all work. When one doesn't work, none work. Seems to be server
side.

Yes to the filters.

Also, now, it seems to be intermittent. Sometimes they show for the Admin,
sometimes they don't.

**Comment by tidusjar** [comment]
That's is strange. If you don't mind, if you are able to get it into a state where they do not appear and then open up the browsers developer tools (usually F12) and go to the console, are there any errors?

**Comment by my95z34** [comment]
Of course now it's working, lol.  Next time it stops I'll check.

Also, not sure if I was clear with my report, but it's only the movies page.  The TV shows page loads fine.

**Comment by tidusjar** [comment]
@my95z34 Any luck?

**Comment by my95z34** [comment]
It's still working normally... Lmao. I guess we can close this and if it
happens again I'll comment with the info you asked for.

I love how it was happening 75% of the time before I report the issue, then
stops. Lol

Thank you though.

**Comment by tidusjar** [comment]
Haha that is usually the case with everything!

Not a problem, I get emails with issues so when you comment on this I will see.

**Comment by my95z34** [comment]
It's happening again. =\

Console shows nothing except the fact that it's navigating to the specific page.

**Comment by my95z34** [comment]
http://imgur.com/lDU4SYW

Also, it may be noted that this is immediately after a reboot of the server.

**Comment by tidusjar** [comment]
Hmmm. That's strange. Browser you are using?

**Comment by my95z34** [comment]
Chrome, on both my Android phone and my computer.

**Comment by Propayne** [comment]
Good afternoon,
I wanted to comment that I am experiencing the same behavior. I have tried the mainstream browsers on OSX, Windows, Android and IOS. 

The service runs on a W7 box and as mentioned before occasionally will show the movie requests, but it doesn't seem to be tied to any event when it happens. I did notice the issue started after a rather large number of requests were submitted back to back (85 of them) in one short session.
